addon/components/dropdown-link.ts

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { scheduleOnce } from '@ember/runloop';
+import { action } from '@ember/object';
+
+export interface DropdownLinkArgs {
```

Comment:
I'd define the other args you are expecting in the template in this interface as well, like I did here:

https://github.com/Gavant/gavant-ember-button-link/blob/master/addon/components/button-link.ts#L13